src/meltano/core/logging/output_logger.py

```diff
@@ -202,7 +202,7 @@ def redirect_logging(self, ignore_errors=()):
         except ignored_errors:  # noqa: WPS329
             raise
         except Exception as err:
-            logger.error(str(err), exc_info=True)
```

Comment:
`exc_info=True` is not too useful here, since it'll point to a `raise RunnerError` if one of the blocks failed.